clusters/build-clusters/01_cluster/machine_config/debug_ctrcfg.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: machineconfiguration.openshift.io/v1
+kind: ContainerRuntimeConfig
+metadata:
+ name: debug
+spec:
+ machineConfigPoolSelector:
+   matchLabels:
+     custom-crio: log-devel-debug
```

Comment:
According to [example](https://github.com/openshift/machine-config-operator/blob/master/docs/ContainerRuntimeConfigDesign.md#example),
we need to

> oc label machineconfigpool worker custom-crio=log-devel-debug

Then it applies all workers from all machinesets.
How can we apply it only to `ms/build01-9hdwj-worker-us-east-1a-debug`?

I guess the MSP will come back.

Reply:
We need to create a custom pool - https://github.com/openshift/machine-config-operator/blob/master/docs/custom-pools.md

Reply:
Added.

Also label with 

https://github.com/openshift/release/blob/b90ae0923aada2e20f4c4c972fed8ad70d3acc18/clusters/build-clusters/01_cluster/machine_config/debug_mcp.yaml#L2-L6